blobit-core/src/test/java/org/blobit/core/cluster/LedgerLifeTest.java

```diff
@@ -154,7 +156,10 @@ public void testNoTablespaces() throws Exception {
                         size());
 
                 // force close all ledgers
-                manager.getBlobManager().closeAllActiveWritersForTests();
+                TestUtils.waitForCondition(() -> {
+                    boolean activeWritersExist = manager.getBlobManager().closeAllActiveWritersForTests();
```

Comment:
We should call closeAllActiveWritersForTests only once and then wait for the actual closing and removal of the writer from the actualWriters list

Reply:
Make sense. Check my updated